src/Command/Image/MissingImagesCommand.php

```diff
@@ -43,11 +43,11 @@ public function __construct(
      *
      * @return void
      */
-    protected function configure()
+    protected function configure(): void
     {
         $this->setDescription('Find images in CoverStore that is not in the image table')
-            ->addOption('vendor-id', null, InputOption::VALUE_OPTIONAL, 'Limit the re-index to vendor with this id number')
-            ->addOption('identifier', null, InputOption::VALUE_OPTIONAL, 'If set only this identifier will be re-index (requires that you set vendor id)');
+            ->addOption('vendor-id', null, InputOption::VALUE_REQUIRED, 'Limit the re-index to vendor with this id number')
+            ->addOption('identifier', null, InputOption::VALUE_REQUIRED, 'If set only this identifier will be re-index (requires that you set vendor id)');
```

Comment:
No sure that this change in requirements make sens.

Reply:
It's should be able to "Find images in CoverStore that is not in the image table" hence we do not have identifier